content/c/concepts/strings/terms/strtok/strtok.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'strtok()'
+Description: 'Breaks a string into a series of tokens using a list of delimiters.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+The **`strtok()`** function breaks a string into a series of tokens using a list of delimiters.
+
+## Syntax
+
+```pseudo
+char *token = strtok(string, delimiters);
```

Comment:
```suggestion
char *token = strtok(string, delimiters);

token = strtok(NULL, delimiters);
```

I feel like adding this line better supports the points made in the paragraph  below.